include/RAJA/RAJA.hxx

```diff
@@ -65,39 +65,35 @@
 
 #include "RAJA/internal/defines.hxx"
 
-#include "RAJA/int_datatypes.hxx"
-#include "RAJA/real_datatypes.hxx"
+#include "RAJA/internal/int_datatypes.hxx"
```

Comment:
Will make it easier for users to pick and choose header files that they want to include in their apps.